OC4IDS_Publisher_Status_Report.ipynb

```diff
@@ -0,0 +1,1796 @@
+{
```

Comment:
<p>Using run_id = 2024-08-02 06:29:25.083245</p><p> </p><p>Cell didn't run, got the error</p><p> </p>

<pre class="ql-syntax" spellcheck="false">File "&lt;ipython-input-20-704129bd1c51&gt;", line 5
    source_id,
    ^
IndentationError: unexpected indent
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Reply:
<p>Fixed the sql magic syntax error</p>